docs/CHANGELOG.asciidoc

```diff
@@ -45,6 +46,198 @@ This section summarizes the changes in each release.
 //[float]
 //=== Known Issues
 
+
+[[release-notes-7.0.0-alpha2]]
+== {kib} 7.0.0-alpha2
+
+coming[7.0.0-alpha2]
+
+[float]
+[[breaking-7.0.0-alpha2]]
+=== Breaking changes
+
+For more details about breaking changes in this release, see 
+<<breaking-changes-7.0, Breaking changes in 7.0>>.
+
+[float]
+[[highlight-7.0.0-alpha2]]
+=== Highlights
+Canvas::
```

Comment:
Let's make the writing of these 3 items parallel:

Canvas now has a template tab in the workpad manager where ...